pool/endpoint.go

```diff
@@ -14,8 +14,72 @@ import (
 	"time"
 
 	"github.com/decred/dcrd/chaincfg/v3"
+	errs "github.com/decred/dcrpool/errors"
 )
 
+var (
```

Comment:
I think all of the miner ID related stuff is good candidate for a new file - potentially `miner_id.go`? Miner ID is a distinct concept to the endpoint concept, so arguably does not belong in `endpoint.go`.

- The ID strings
- `minerIDPair`
- `newMinerIDPair`
- `generateMinerIDs`
- `identifyMiner`